packages/erlang/build.sh

```diff
@@ -11,7 +12,12 @@ TERMUX_PKG_DEPENDS="openssl, ncurses, zlib"
 TERMUX_PKG_NO_STATICSPLIT=true
 TERMUX_PKG_HOSTBUILD=true
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--without-javac --with-ssl=${TERMUX_PREFIX} --with-termcap"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+--without-javac
+--with-ssl=${TERMUX_PREFIX}
+--with-termcap
+erl_xcomp_sysroot=${TERMUX_PREFIX}
```

Comment:
Two dashes unneeded here?

Reply:
Unneeded because it's an autoconf cache variable.